### YamlMime:UniversalReference
items:
  - uid: '@azure/servicefabric.TestErrorChaosEvent'
    name: TestErrorChaosEvent
    fullName: TestErrorChaosEvent
    children:
      - '@azure/servicefabric.TestErrorChaosEvent.kind'
      - '@azure/servicefabric.TestErrorChaosEvent.reason'
      - '@azure/servicefabric.TestErrorChaosEvent.timeStampUtc'
    langs:
      - typeScript
    type: interface
    summary: |-
      <span data-ttu-id="72b05-101">Beschreibt ein Chaos-Ereignis, das generiert wird, tritt ein unerwartetes Ereignis in der Chaos-Engine.</span><span class="sxs-lookup"><span data-stu-id="72b05-101">Describes a Chaos event that gets generated when an unexpected event occurs in the Chaos engine.</span></span>
      <span data-ttu-id="72b05-102">Chaos gefunden aufgrund der Momentaufnahme des Clusters inkonsistent, während eine Entität Fehler, z. B., dass die Entität bereits fehlerhaft wurde – Was wäre ein unerwartetes Ereignis.</span><span class="sxs-lookup"><span data-stu-id="72b05-102">For example, due to the cluster snapshot being inconsistent, while faulting an entity, Chaos found that the entity was already faulted -- which would be an unexpected event.</span></span>
    package: '@azure/servicefabric'
  - uid: '@azure/servicefabric.TestErrorChaosEvent.kind'
    name: kind
    fullName: kind
    children: []
    langs:
      - typeScript
    type: property
    summary: <span data-ttu-id="72b05-103">Polymorphe Diskriminator</span><span class="sxs-lookup"><span data-stu-id="72b05-103">Polymorphic Discriminator</span></span>
    syntax:
      content: 'kind: "TestError"'
      return:
        type:
          - '"TestError"'
    package: '@azure/servicefabric'
  - uid: '@azure/servicefabric.TestErrorChaosEvent.reason'
    name: reason
    fullName: reason
    children: []
    langs:
      - typeScript
    type: property
    summary: '<span data-ttu-id="72b05-104">Beschreibt, warum TestErrorChaosEvent generiert wurde.</span><span class="sxs-lookup"><span data-stu-id="72b05-104">Describes why TestErrorChaosEvent was generated.</span></span> <span data-ttu-id="72b05-105">Chaos wird beispielsweise versucht, eine Partition einen Fehler, aber feststellt, dass die Partition nicht mehr an Fehlertoleranz zu erzielen ist, und klicken Sie dann eine TestErrorEvent generiert wird, mit dem Grund an, die besagt, dass die Partition nicht fehlertolerant ist.</span><span class="sxs-lookup"><span data-stu-id="72b05-105">For example, Chaos tries to fault a partition but finds that the partition is no longer fault tolerant, then a TestErrorEvent gets generated with the reason stating that the partition is not fault tolerant.</span></span>'
    optional: true
    syntax:
      content: 'reason?: undefined | string'
      return:
        type:
          - undefined | string
    package: '@azure/servicefabric'
  - uid: '@azure/servicefabric.TestErrorChaosEvent.timeStampUtc'
    name: timeStampUtc
    fullName: timeStampUtc
    children: []
    langs:
      - typeScript
    type: property
    summary: '<span data-ttu-id="72b05-106">Der UTC-Zeitstempel, dieses Chaos-Ereignis generiert wurde.</span><span class="sxs-lookup"><span data-stu-id="72b05-106">The UTC timestamp when this Chaos event was generated.</span></span>'
    syntax:
      content: 'timeStampUtc: Date'
      return:
        type:
          - Date
    package: '@azure/servicefabric'